package.json

```diff
@@ -44,7 +44,6 @@
     "istanbul-lib-coverage": "^1.0.0",
     "jasmine-reporters": "^2.2.0",
     "jquery": "^3.2.1",
-    "jsdom": "^9.12.0",
```

Comment:
any reason for it to be specified down here as well?

Reply:
Nope I guess, workspaces should handle this.

Reply:
yeah, it's hoisted, and I didn't see any require of it outside of `jest-environment-jsdom`

Reply:
If it works, that's fine. We used to have a test depending on jsdom, but that was all pre workspaces.